overhangio/tutor
# fix: clean tutor nightly from olive-specific changes

This solves the issue #767 by providing the following fixes:
1. Replace `open-release/olive.1` tag with the expected `master` branch.
2. Remove a patch for 'circuit schematic builder problem' that shouldn't be present in tutor nightly since it's already merged into `edx-platform:master` branch.

**Testing:**
1. Install this branch of tutor nightly.
2. run `tutor config save` to generate an environment if not already generated.
3. run `tutor images build openedx`. It should now work. 

It would be better if this PR covers any other release-specific changes that should be cleaned too.
FYI @regisb
